vector/src/main/java/im/vector/app/core/pushers/VectorMessagingReceiver.kt

```diff
@@ -166,6 +169,19 @@ class VectorMessagingReceiver : MessagingReceiver() {
         }
     }
 
+    override fun onReceive(context: Context, intent: Intent) {
+        // Injections happens here
+        super.onReceive(context, intent)
+
+        // if it is from FirebaseReceiver, then the token has been rejected
```

Comment:
What do you mean by `rejected`?

Reply:
It was not a token saved by the unifiedpush connector so it was rejected. It is outdated now.